tokio/tests/sync_notify.rs

```diff
@@ -225,3 +226,42 @@ fn test_waker_update() {
 
     assert!(future.is_woken());
 }
+
+// tokio-rs/tokio#5396
+#[tokio::test(flavor = "multi_thread")]
```

Comment:
This test takes around 0.5 second to complete. I think there should be a loom test instead, but I was unable to create one which would fail on master...

Reply:
It is now replaced with a loom test which fails on master.